apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -224,6 +224,18 @@ public override UpdateRowSource UpdatedRowSource
         /// </summary>
         public QueryOptions QueryOptions { get; set; }
 
+        // Request options for this command. Only the relevant properties are publicly exposed.
+        internal RequestOptions _requestOptions = RequestOptions.Empty;
+
+        /// <summary>
+        /// The statement tag to send to Cloud Spanner for this command.
+        /// </summary>
+        public string RequestTag
```

Comment:
Yeah, I understand what you mean @amanda-tarafa. But I think @olavloite is right here in the sense that there will be UI that shows the stats and it would then be confusing for customers when they can't filter on a "Command Tag".

Reply:
@syeduguri would you be able to give some information on how users will be able to see this data? Would it be too confusing if we called it a CommandTag?

Reply:
Yes. CommandTag will be confusing. I think we can rename it as just "Tag".

Reply:
I've renamed both `RequestTag` and `TransactionTag` to `Tag`.